include/fizzy/fizzy.h

```diff
@@ -169,6 +169,18 @@ void fizzy_free_module(const FizzyModule* module);
 /// @note  Input module is not modified neither in success nor in failure case.
 const FizzyModule* fizzy_clone_module(const FizzyModule* module);
 
+/// Get number of types defined in the module.
+///
+/// @param  module    Pointer to module. Cannot be NULL.
+uint32_t fizzy_get_type_count(const FizzyModule* module);
+
+/// Get type defined in the module.
+///
+/// @param  module      Pointer to module. Cannot be NULL.
+/// @param  type_idx    Type index. Behaviour is undefined, if index is not valid according
```

Comment:
I think the comma is not needed:
```suggestion
/// @param  type_idx    Type index. Behaviour is undefined if index is not valid according
```